web/models/machine.ex

```diff
@@ -22,11 +23,12 @@ defmodule CodeStats.Machine do
     |> cast(params, [:name])
     |> name_validations()
     |> put_change(:api_salt, generate_api_salt())
+    |> put_change(:active, true)
   end
 
   def update_changeset(data, params \\ %{}) do
     data
-    |> cast(params, [:name])
+    |> cast(params, [:name, :active])
```

Comment:
It might be better to have a separate changeset for updating the active status. Now this changeset requires both `:name` and `:active` and that may lead to problems and inadvertent changes of the other value when the other is changed and the other is left unset.